crates/ruff_python_ast/src/strings.rs

```diff
@@ -1,26 +1,33 @@
-use ruff_python_stdlib::str::{
-    SINGLE_QUOTE_PREFIXES, SINGLE_QUOTE_SUFFIXES, TRIPLE_QUOTE_PREFIXES, TRIPLE_QUOTE_SUFFIXES,
-};
-
 /// Strip the leading and trailing quotes from a docstring.
 pub fn raw_contents(contents: &str) -> &str {
-    for pattern in TRIPLE_QUOTE_PREFIXES {
+    for pattern in ruff_python_stdlib::str::TRIPLE_QUOTE_PREFIXES
```

Comment:
What's the reason that `raw_contents` is defined in the AST crate vs in the `stdlib` crate? Defining all symbols in the same crates would have the advantage that it may not be necessary for `SINGLE_QUOTE_PREFIXES` to be public. 

Maybe the more fundamental question is what's the difference between the `stdlib` and `ast` crates and is it worth having both or should we merge them into a single `python_syntax` crate? If not, could we add a README to each create (or documentation to `lib.rs`) explaining the intended purpose of each crate.

Reply:
I think these should actually be moved into the `ast` crate.

The `stdlib` crate was intended to serve as roughly 1:1 reimplementations of functionality that you typically get from the Python standard library (e.g., lists of keywords, checks for whether something is a valid identifier, lists of standard library modules, and so on).